src/components/proposal/ProposalSummary.tsx

```diff
@@ -74,12 +70,20 @@ function ProposalReview({ readonly, confirm }: ProposalSummaryProps) {
     !allStepsComplete ||
     proposal.submitted;
 
+  function SubmitButtonMessage() {
+    if (
+      proposal.status != null &&
+      proposal.status.shortCode.toString() == 'EDITABLE_SUBMITTED'
```

Comment:
I realised that this will never be shown, because when a user modifies a proposal in the `EDITABLE_SUBMITTED` state, the proposal is saved, not re-submitted.

I think we instead want to somehow look ahead to the upcoming proposal workflow statuses and show this if one of them is `EDITABLE_SUBMITTED`.